pwndbg/gdblib/kernel/__init__.py

```diff
@@ -76,6 +78,15 @@ def kversion() -> str:
     return pwndbg.gdblib.memory.string(version_addr).decode("ascii").strip()
 
 
+@requires_debug_syms()
+@pwndbg.lib.memoize.reset_on_start
+def krelease() -> Tuple[int, ...]:
+    match = re.search(r"Linux version (\d+)\.(\d+)(?:\.(\d+))?", kversion())
+    if match:
+        return tuple([int(x) for x in match.groups() if x])
```

Comment:
nitpick: I don't think the nested list is necessary. You should just be able to do: `tuple(int(x) for x in match.groups() if x)`